Controller/Adminhtml/Password/ChangePwdPost.php

```diff
@@ -32,23 +34,33 @@ class ChangePwdPost extends Action implements HttpPostActionInterface
      */
     protected $encryptor;
 
+    /**
+     * @var AddressRegistry
+     */
+    private $addressRegistry;
+
     /**
      * ChangePwdPost constructor.
      *
      * @param Context $context
      * @param CustomerRepositoryInterface $customerRepository
      * @param CustomerRegistry $customerRegistry
      * @param EncryptorInterface $encryptor
+     * @param AddressRegistry|null $addressRegistry
      */
     public function __construct(
         Context $context,
         CustomerRepositoryInterface $customerRepository,
         CustomerRegistry $customerRegistry,
-        EncryptorInterface $encryptor
+        EncryptorInterface $encryptor,
+        ?AddressRegistry $addressRegistry = null
     ) {
         $this->customerRepository = $customerRepository;
         $this->customerRegistry = $customerRegistry;
         $this->encryptor = $encryptor;
+        $objectManager = ObjectManager::getInstance();
+        $this->addressRegistry = $addressRegistry
+            ?: $objectManager->get(AddressRegistry::class);
```

Comment:
We don't need to keep $objectManager = ObjectManager::getInstance(); defined as a constructor, just remove it.

So in this case the

```
$this->addressRegistry = $addressRegistry
            ?: ObjectManager::getInstance()->get(AddressRegistry::class);
```            

just enough.